stanford/playbooks/deploy.yml

```diff
@@ -45,9 +47,9 @@
         requirements: requirements.txt
       tags:
         - requirements
-    - name: Checkout configuration code for each user
+    - name: Checkout configuration code for each user (team)
       git:
-        remote: "{{ item.name }}"
+        remote: origin
```

Comment:
This had been mistakenly adding the Stanford-Online fork under a remote named for the user.
The original intent was to add personal forks and/or the common team remote.

Now, we clone the repo from Stanford-Online,
then add a remote for your personal fork.